WordPress/gutenberg
# FontSizerPicker: Improve docs for default units

## What?

Clarify in the docs/code for FontSizerPicker that there is a default value for the `units` prop.

## Why?

I noticed when reviewing #60607 that this was unclear.

## Testing Instructions

- In the "With Units" story for FontSizePicker, click on the "Set custom size" button to reveal the number input. The unit dropdown should still show the default units.
- Passing a custom unit array to the `units` prop should still work, which is asserted by [this unit test](https://github.com/WordPress/gutenberg/blob/6527034b9e00c4a1297f5fcf5d11e5a10dfbaa02/packages/components/src/font-size-picker/test/index.tsx#L621).
